src/components/billingRecord/IFXBillingRecords.vue

```diff
@@ -75,6 +75,9 @@ export default {
     isDecimalFacility(facility) {
       return ['Research Computing Storage'].includes(facility.name)
     },
+    isFacilityWithDates(facility) {
+      return ['Center for Brain Science Neuroimaging'].includes(facility.name)
+    },
```

Comment:
Rather than do this, can you just make `showDates` a prop that is passed through to `IFXBillingRecordList`? This is done with `allowDownloads` and lets the facility app decide if it wants to have dates displayed.

Reply:
All set